github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml

```diff
@@ -195,3 +195,45 @@ postsubmits:
         resources:
           requests:
             memory: "8Gi"
+  - name: push-cdi-goveralls
+    cluster: kubevirt-prow-control-plane
+    annotations:
+      testgrid-create-test-group: "false"
+    always_run: true
+    optional: true
+    skip_reporting: true
```

Comment:
Same for `skip_reporting: true`  - it would be good to have the result of this reported alongside the commit.

Reply:
Right. Fixed :)